rest-api/src/main/java/com/example/rest_api/controllers/BookmarkController.java

```diff
@@ -0,0 +1,100 @@
+package com.example.rest_api.controllers;
+
+import com.example.rest_api.entities.Bookmark;
+import com.example.rest_api.repositories.BookmarkRepository;
+import lombok.extern.java.Log;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/api/v1/bookmarks")
+@Log
+public class BookmarkController {
+
+  @Autowired BookmarkRepository bookmarkRepository;
```

Comment:
Please make use of constructor based injection instead of field based injection

Reply:
Updated it.